README.md

```diff
@@ -631,6 +631,22 @@ curl -X 'POST' 'http://127.0.0.1:8080/v1/query' -H 'accept: application/json' -H
 
 > You can use the `/v1/streaming_query` (with the same parameters) to get the streaming response (SSE/HTTP chunking). By default, it streams text, but you can also yield events as JSONs via additional `"media_type": "application/json"` parameter in the payload data.
 
+> In devel environment where authentication module is set to `noop` it is possible to specify optional query parameter `user_id` with user identification. This parameter can be set to any value, but currently it is preferred to use UUID.
+
+
+### Validation if the logged-in user is authorized to access service
+
+To validate if the logged-in user is authorized to access the service, the following REST API call can be utilized:
+
+```sh
+curl -X 'POST' 'http://localhost:8080/authorized' -H 'accept: application/json'
+```
+
+For authorized users, 200 OK is returned, otherwise `401 Missing or invalid credentials provided by client` or `403 User is not authorized` HTTP codes can be returned.
+
+> In devel environment where authentication module is set to `noop` it is possible to specify optional query parameter `user_id` with user identification. This parameter can be set to any value, but currently it is preferred to use UUID.
```

Comment:
are we meant to be repeating this paragraph?

Reply:
it is note/comment for different endpoint. And just three endpoints accepts `user_id` so dunno how to make it more modular. WDYT @JoaoFula ?

Reply:
resolved via PMs